scylla-cdc-base/src/main/java/com/scylladb/cdc/model/worker/RawChange.java

```diff
@@ -124,12 +124,56 @@ default ByteBuffer getAsUnsafeBytes(String columnName) {
      */
     ByteBuffer getAsUnsafeBytes(ChangeSchema.ColumnDefinition columnDefinition);
 
+    /**
+     * Returns the boolean value of the deleted column for the given column name.
+     * <p>
+     * This method returns the value of the <code>cdc$deleted_</code> column for
+     * the given column name. It is only relevant for the clustering columns
```

Comment:
Actually, my comment was wrong, and there __is__ a cdc$deleted_ column for clustering columns of the base table. Sorry for the confusion.

Reply:
I was wrong again... When I previously tested if clustering columns had the cdc$deleted_ field, I accidentally made a regular column instead of a clustering column.

Clustering columns do not seem to have the corresponding cdc$deleted_ column. Here is output from cqlsh as a proof:

```cql
cqlsh> create table ks1.tbl (pk int, ck int, v int, s int static, PRIMARY KEY (pk, ck)) with cdc = {'enabled': true};
cqlsh> DESCRIBE TABLE ks1.tbl_scylla_cdc_log ;

CREATE TABLE ks1.tbl_scylla_cdc_log (
    "cdc$stream_id" blob,
    "cdc$time" timeuuid,
    "cdc$batch_seq_no" int,
    "cdc$deleted_s" boolean,
    "cdc$deleted_v" boolean,
    "cdc$end_of_batch" boolean,
    "cdc$operation" tinyint,
    "cdc$ttl" bigint,
    ck int,
    pk int,
    s int,
    v int,
    PRIMARY KEY ("cdc$stream_id", "cdc$time", "cdc$batch_seq_no")
) WITH CLUSTERING ORDER BY ("cdc$time" ASC, "cdc$batch_seq_no" ASC)
    AND bloom_filter_fp_chance = 0.01
    AND caching = {'enabled': 'false', 'keys': 'NONE', 'rows_per_partition': 'NONE'}
    AND comment = 'CDC log for ks1.tbl'
    AND compaction = {'class': 'TimeWindowCompactionStrategy', 'compaction_window_size': '60', 'compaction_window_unit': 'MINUTES', 'expired_sstable_check_frequency_seconds': '1800'}
    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}
    AND crc_check_chance = 1.0
    AND dclocal_read_repair_chance = 0.0
    AND default_time_to_live = 0
    AND gc_grace_seconds = 0
    AND max_index_interval = 2048
    AND memtable_flush_period_in_ms = 0
    AND min_index_interval = 128
    AND read_repair_chance = 0.0
    AND speculative_retry = '99.0PERCENTILE';
```

So, only regular and static columns have the cdc$deleted_ field.

Reply:
Changed the documentation from: "clustering column" to "regular column" (and a few usage of "clustering column" terms in the added tests).